src/main/java/com/parasoft/findings/jenkins/internal/rules/RuleDocumentationStorage.java

```diff
@@ -89,7 +89,7 @@ private String readFromLocal(String ruleDocLocation)
         if (localFile != null) {
             try {
                 return FileUtil.readFile(localFile, IStringConstants.UTF_8);
-            } catch (IOException e) {
+            } catch (IOException e) { // parasoft-suppress OWASP2021.A9.LGE "This is expected. Reason: Do not fail the build when processing coverage reports."
```

Comment:
Please review all the messages. For example, in this case, it appears to be processing a static analysis report instead of a coverage report.